stdlib/mexpr/ast-smap-sfold-tests.mc

```diff
@@ -190,7 +193,7 @@ in
 let countNodes = bottomUp (sfold_Expr_Expr addi 1) in
 
 utest bottomUp identity tmVarX with tmVarX in
-utest bottomUp cInt2cChar tmRecordI with tmRecordC in
+utest bottomUp cInt2cChar tmRecordI with tmRecordC using eqExpr in
```

Comment:
How come this test case needed to use `eqExpr`?

Reply:
Not sure, I guess it has to do something with the new map data type used to hold record fields. The default utest (without `using`) failed at least.